modyn/config/schema/pipeline/sampling/downsampling_config.py

```diff
@@ -147,6 +147,9 @@ class ILTrainingConfig(TrainingConfig):
     initial_model: Literal["random"] = "random"
     initial_model_id: Literal[None] = None
     checkpointing: NoCheckpointingConfig = Field(default=NoCheckpointingConfig())
+    drop_last_batch: bool = Field(
+        default=False, description="Whether to drop the last batch if it is smaller than the batch size."
+    )
```

Comment:
The default value for the normal training config is `True` because we want to drop the last batch.

Here this is for the training for an il model (which is used to guide the data selection for the main model). We want to disable last batch dropping because we don't use selection strategies on il model so the previous exception triggered by not dropping last batch is avoided. Also we want to use as much data s possible.